onnxruntime/core/framework/provider_bridge_ort.cc

```diff
@@ -510,6 +510,9 @@ struct ProviderLibrary {
       return;
 
 #if defined(_WIN32) && !defined(_OPENMP)
+#if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP)
+#error Unsupported configuration
```

Comment:
can you put a comment on why this is unsupported?

Reply:
I'm not too sure why this DNNL/OMP workaround is needed in the first place, I'm asking ORT here: https://github.com/microsoft/onnxruntime/pull/3978#issuecomment-667342817 . Maybe there's a way to make it work without using GetModuleHandleExA